test/askContainer/treeViews/treeViewProviders/helpViewProvider.test.ts

```diff
@@ -0,0 +1,157 @@
+import * as vscode from "vscode";
```

Comment:
TreeView `helpViewProvider` and webView `deploySkillWebview` are both 100% coverage. 👍